packages/xpath/src/xforms/XFormsItextTranslations.ts

```diff
@@ -156,51 +162,43 @@ export class XFormsItextTranslations<T extends XPathNode> implements XFormsItext
 	}
 
 	/**
+	 * Retrieves all translation value elements for a given Itext in the active language.
+	 *
 	 * @package
 	 *
-	 * Here, "default" is meant as more precise language than "regular" as
-	 * {@link https://getodk.github.io/xforms-spec/#languages | specified by ODK XForms}. In other words, this is equivalent to the following hypothetical XPath pseudo-code (whitespace added to improve structural clarity):
+	 * This method fetches the `text` element matching `itextID` and returns its child `value` elements,
+	 * which may have an optional `@form` attribute.
+	 *
+	 * The operation is conceptually similar to the following XPath query:
 	 *
 	 * ```xpath
-	 * string(
-	 *   imaginary:itext-translation(
-	 *     xpath3-fn:environment-variable('activeLanguage')
-	 *   )
-	 *     /text[@id = $itextID]
-	 *       /value[not(@form)]
+	 * imaginary:itext-translation(
+	 *   xpath3-fn:environment-variable('activeLanguage')
 	 * )
+	 *   /text[@id = $itextID]
+	 *     /value
 	 * ```
 	 *
 	 * Or alternately:
 	 *
 	 * ```xpath
-	 * string(
-	 *   imaginary:itext-root()
-	 *     /translation[@lang = xpath3-fn:environment-variable('activeLanguage')]
-	 *       /text[@id = $itextID]
-	 *         /value[not(@form)]
-	 * )
+	 * imaginary:itext-root()
+	 *   /translation[@lang = xpath3-fn:environment-variable('activeLanguage')]
+	 *     /text[@id = $itextID]
+	 *       /value
 	 * ```
-	 *
-	 * @todo The above really feels like it adds some helpful clarity to how `jr:itext()` is designed to work, and the kinds of structures, state and input involved. Since there's already some discomfort around that API as specified, it's worth considering
+	 * Returns an array of `XFormsItextTranslationValueElement<T>` or `null` if no matching `text` element is found.
 	 */
-	getDefaultTranslationText(itextID: string): string {
+	getTranslationValues(itextID: string): Array<XFormsItextTranslationValueElement<T>> | null {
 		const textElement = this.getTranslationTextElement(itextID);
 
 		if (textElement == null) {
-			return '';
-		}
-
-		const { domProvider } = this;
-
-		for (const valueElement of domProvider.getChildrenByLocalName(textElement, 'value')) {
-			//
-			if (!domProvider.hasLocalNamedAttribute(valueElement, 'form')) {
-				return domProvider.getNodeValue(valueElement);
```

Comment:
Fact: This is where the actual XPath query happens; it returned the first `value` nodes that didn't have the `form` attribute. 
Now, it returns all value nodes, and they are evaluated in the engine (see `createTextRange` file)